src/lib/serviceRoutines/postSubscribeContext.cpp

```diff
@@ -82,7 +84,47 @@ std::string postSubscribeContext
     return answer;
   }
 
-  TIMED_MONGO(ciP->httpStatusCode = mongoSubscribeContext(&parseDataP->scr.res, &scr, ciP->tenant, ciP->uriParam, ciP->httpHeaders.xauthToken, ciP->servicePathV));
+
+  //
+  // String filter in URI param 'q' ?
```

Comment:
NGSIv1 subscriptions doesn't support q. Note that q as URI param has been only defined for `GET /v2/entities` operation. Thus I guess that all this new code should be rolled-back.